text/0097-log-data-model.md

```diff
@@ -167,7 +172,7 @@ SpanId         |Request span id.
 TraceFlags     |W3C trace flag.
 SeverityText   |The severity text (also known as log level).
 SeverityNumber |Numerical value of the severity.
-ShortName      |Short event identifier.
+Name           |Short event identifier.
```

Comment:
```suggestion
Name           |Event identifier.
```

Reply:
I think it is useful to keep "short" qualifier short to emphasize that people need to keep it small.

Reply:
Right! Got it. Seems good then.